go.mod

```diff
@@ -11,27 +11,19 @@ require (
 	github.com/spf13/cobra v1.7.0
 	github.com/spf13/pflag v1.0.5
 	github.com/texttheater/golang-levenshtein/levenshtein v0.0.0-20200805054039-cae8b0eaed6c
-	k8s.io/api v0.25.6
-	k8s.io/apimachinery v0.25.6
-	k8s.io/cli-runtime v0.25.6
-	k8s.io/client-go v0.25.6
-	k8s.io/klog/v2 v2.80.1
-	k8s.io/kubectl v0.25.6
-	k8s.io/utils v0.0.0-20221012122500-cfd413dd9e85
+	k8s.io/api v0.27.3
+	k8s.io/apimachinery v0.27.3
+	k8s.io/cli-runtime v0.27.3
+	k8s.io/client-go v0.27.3
```

Comment:
What was our plan here? Shouldn't we keep that in sync with what we have in [Shipwright Build](https://github.com/shipwright-io/build/blob/4377cb756018282a02877963063dbd2179049cb2/go.mod#L19-L23), i. e. `v0.25`-ish?

Reply:
I think we are still working on deciding what version to use, this was just the repository that I picked to try the upgrade with. I would like to use v0.27.3 since it won't go EOM as soon and it aligns with the version that we are using in OpenShift.